sw/device/tests/dif/dif_rstmgr_unittest.cc

```diff
@@ -106,9 +106,9 @@ TEST_F(ResetIsLockedTest, Success) {
     // peripheral is locked. One by one we check every peripheral, by setting
     // all bits high apart from the peripheral under test that is indexed
     // by `bit_index`.
-    uint32_t bit_locked = bitfield_bit32_write(
-        std::numeric_limits<uint32_t>::max(), bit_index, false);
-    EXPECT_READ32(RSTMGR_SW_RST_REGEN_REG_OFFSET, bit_locked);
```

Comment:
I think it is better to test the new functionality in `sw/device/lib/testing/mock_mmio_test.cc` similar how the other `EXPECT_WRITE*` family functions are tested.  I guess the thing to do, would be to to add a `TEST_F(MockMmioTest, SomeAppropriateName)` to test the new functionality.

Then when this PR is merged, a good follow-up would be to fixup the relevant places in `dif_rstmgr_unittest.cc`.